examples/docking-protein-homotrimer/docking-protein-homotrimer.cfg

```diff
@@ -64,8 +64,8 @@ c3sym_end3_1="128"
 c3sym_seg3_1="C"
 
 
-#[caprieval]
-#reference_fname = 'data/1qu9_ABC.pdb'
+[caprieval]
```

Comment:
`caprieval` is not fully compatible with multi-body, is this related to the `emscoring` issue? if not its best to remove this from the PR